src/adapters/CartAdapter/index.js

```diff
@@ -93,18 +96,16 @@ class CartAdapter {
           (instance) => instance.form.dataset.name == "cart"
         );
 
-        if (cartInstance == null) {
-          return;
-        }
-
-        const cartItems = JSON.parse(
-          localStorage.getItem("cart") || '["Empty Cart"]'
-        );
         const defaultFilter = new Set();
-
-        cartItems.forEach((item) => {
-          defaultFilter.add(item);
-        });
+        const cartItems = this.listCartItems();
+
+        if (cartItems.length > 0) {
+          cartItems.forEach((item) => {
+            defaultFilter.add(item.slug);
+          });
+        } else {
+          defaultFilter.add("EMPTY_CART");
```

Comment:
Hmm question krub. What would happen if we don't add `EMPTY_CART` to the filter? Same goes for `EMPTY_SUGGESTION` 🤔

Reply:
@teamdisc If no filter is added, the CMS Filter will display all records as if no filter were selected. Therefore, we need to include an empty identifier that is distinct from any item name or slug. This will make the CMS filter return an empty list.